### PR TITLE
Feature/pelagos 3890 size erddap button

### PR DESCRIPTION
**Pull request by fightingtexasaggie**
I got rid of the images of buttons and replaced with buttons, but in loading, multiple classes get applied and they go throw 2 different themes. It's functional, but beyond ugly.

### CODE REVIEW

**Comment by praneethpr** [comment]
Remove the divs and the <br> tags, the buttons should be in the same row

**Comment by fightingtexasaggie** [comment]
See what y'all think all on a single line. The first 2 buttons have routes associated with them, the erddap one does not so I had to get add a data attribute in the template to pass to the js. What I don't like is how the button theme is applied late, and you can see the buttons transition. I guess I could hide them until document.ready to prevent this nastiness.

**Comment by praneethpr** [comment]
> See what y'all think all on a single line. The first 2 buttons have routes associated with them, the erddap one does not so I had to get add a data attribute in the template to pass to the js. What I don't like is how the button theme is applied late, and you can see the buttons transition. I guess I could hide them until document.ready to prevent this nastiness.

For those download buttons, we can use font-awesome icons which will give it a better appearance ?

**Comment by mickel1138** [comment]
I not sure if I like the plain buttons, they are not very prominent.

File: templates/Dataland/summary.html.twig [hunk 1]
```diff
@@ -14,24 +14,23 @@
...
+                            <button id="downloaddsden" data-download-url="{{dataset.datasetSubmission.erddapurl}}" >Download Data</button>
                     {% endif %}
                     {% if dataset.datasetStatus == constant('App\\Entity\\Dataset::DATASET_STATUS_ACCEPTED') %}
-                        <img width="40px" height="40px" id="metadatadl" src="{{ asset('images/xmlc.png') }}">
+                            <button id="metadatadl" data-download-url="{{dataset.datasetSubmission.erddapurl}}" >XML Metadata</button>
                     {% else %}
-                        <img width="40px" height="40px" id="metadatadl-dis" src="{{ asset('images/xmlc-bw.png') }}">
+                            <button id="downloadds" data-download-url="{{dataset.datasetSubmission.erddapurl}}" >XML Metadata</button>
                     {% endif %}
                     {% set TEMPORYDISABLED = true %}
```
Comment (mickel1138) [inline templates/Dataland/summary.html.twig]: This line can go.

Comment (mickel1138) [reply templates/Dataland/summary.html.twig]: No, it can stay for now.

**Pull request merged**